api/src/test/java/org/pmiops/workbench/interceptors/AuthInterceptorTest.java

```diff
@@ -47,95 +59,114 @@ public void handle() {}
   public void handle() {}
 }
 
+@RunWith(SpringRunner.class)
 public class AuthInterceptorTest {
 
   private static final long USER_ID = 123L;
 
-  @Mock private UserInfoService userInfoService;
-  @Mock private FireCloudService fireCloudService;
-  @Mock private UserDao userDao;
+  @MockBean private UserInfoService userInfoService;
+  @MockBean private FireCloudService fireCloudService;
+  @MockBean private UserDao userDao;
+  @MockBean private UserService userService;
+  @MockBean private DevUserRegistrationService devUserRegistrationService;
+
+  private static WorkbenchConfig workbenchConfig;
+
   @Mock private HttpServletRequest request;
   @Mock private HttpServletResponse response;
   @Mock private HandlerMethod handler;
-  @Mock private UserService userService;
+  private DbUser user;
 
   @Rule public MockitoRule mockitoRule = MockitoJUnit.rule();
 
-  private AuthInterceptor interceptor;
-  private DbUser user;
+  @Autowired private AuthInterceptor interceptor;
+
+  @TestConfiguration
+  @Import({AuthInterceptor.class})
+  static class Configuration {
+    @Bean
+    @Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
+    public WorkbenchConfig getWorkbenchConfig() {
+      return workbenchConfig;
+    }
+  }
 
   @Before
   public void setUp() {
-    WorkbenchConfig workbenchConfig = new WorkbenchConfig();
+    workbenchConfig = WorkbenchConfig.createEmptyConfig();
     workbenchConfig.googleDirectoryService = new GoogleDirectoryServiceConfig();
     workbenchConfig.googleDirectoryService.gSuiteDomain = "fake-domain.org";
     workbenchConfig.auth = new AuthConfig();
     workbenchConfig.auth.serviceAccountApiUsers = new ArrayList<>();
     workbenchConfig.auth.serviceAccountApiUsers.add("service-account@appspot.gserviceaccount.com");
-    this.interceptor =
-        new AuthInterceptor(
-            userInfoService, fireCloudService, Providers.of(workbenchConfig), userDao, userService);
-    this.user = new DbUser();
+    user = new DbUser();
```

Comment:
You could make this a bean I think.

Reply:
Not sure why?

Reply:
Oh  if this class doesn't use a user provider then I guess t that doesn't make sense.